elections/tools/README.md

```diff
@@ -0,0 +1,37 @@
+
+# Election Tools
+
+This directory contains the tools required to generate the kata electorate
+
+## generate_electorate.py
+
+This tool need the following python libraries pytz github3.py pyyaml.  To
```

Comment:
s/need/needs/

Maybe make the requirements a bullet list:

- pytz
- github3.py
- pyyaml